modules/ztp-adding-worker-nodes.adoc

```diff
@@ -0,0 +1,196 @@
+// Module included in the following assemblies:
+// Epic CNF-5335 (4.11), Story TELCODOCS-643
+// scalability_and_performance/ztp-deploying-disconnected.adoc
+
+:_content-type: PROCEDURE
+[id="ztp-additional-worker-sno-proc"]
```

Comment:
Why don't your IDs have _{context}? All modules should use _{context} in the IDs.